api/docs/release.dox

```diff
@@ -143,8 +143,8 @@ changes:
    #dynamorio::drmemtrace::TRACE_MARKER_TYPE_UNCOMPLETED_INSTRUCTION was added
    to indicate an uncompleted instruction was removed. The value of the marker
    is the encoding of the removed instruction up to a pointer's length. Added
-   #OFFLINE_FILE_VERSION_RETIRED_INSTRUCTIONS_ONLY to increase the trace version
-   for drmemtraces with uncompleted instructions removed.
+   #dynamorio::drmemtrace::TRACE_ENTRY_VERSION_RETIRED_INSTRUCTIONS_ONLY to increase the
```

Comment:
This is the 11.1 changelist.  We need an entry up in the post-11.1 list on line 129.  For here we should probably explain that the offline version was incremented accidentally and later corrected.